docs/user/cli/ingest.rst

```diff
@@ -49,29 +49,27 @@ The ``--converter`` argument may be any of the following:
 * A converter configuration string
 * The name of a file containing a converter configuration
 
-See :ref:`cli_converter_conf` for more details on specifying the converter.
-
-If a converter is not specified, GeoMesa will attempt to extract a schema from the input files. Currently
-this only works in a very limited fashion - the input must be a Shapefile, self describing Avro, or
-delimited text (TSV, CSV) with a header row that defines the attribute name and type of each column. For example::
```

Comment:
If one still has a file with a header, what happens under inferencing?  Does it still work as before?

Reply:
no, it won't use the types from the header - but you can pull out the header and pass it as the sft spec and it will use that